src/lib/locales/ch.json

```diff
@@ -51,52 +51,56 @@
 				"tools": "Tools"
 			}
 		}
 	},
 	"team": {
 		"nav_item": "Team",
 		"title": "Üses Team",
-		"body": "Mir si Cloud Software Engineers vor erste Stund mit Bruefserfahrig sowohl i Grossundernähme als ou i Start-ups. Mir si stolz uf üsi Fähigkeite als kompetänti Berater*inne, ifühlsami Coaches und erfahreni Coder*inne. Interessiert üs z träffe? De chömet eifach ungeniert über Social Media uf üs zue, träffet üs amne Meetup ir Nechi oder nützet ds Kontaktformular unde.",
+		"body": "Mir si Cloud Software Engineers vor erste Stund mit Bruefserfahrig sowohl i Grossundernähme als ou i Start-ups. Mir si stolz uf üsi Fähigkeite als kompetänti Berater*inne, ifühlsami Coaches u erfahreni Coder*inne. Interessiert üs z träffe? De chömet eifach ungeniert über Social Media uf üs zue, träffet üs amne Meetup ir Nechi oder nützet ds Kontaktformular unde.",
 		"mathis": {
 			"catchphrase": "Keeper of Simple",
-			"body": "Softwarearchitekt und Mitgründer. Spezialisert uf Microservices, Container-Technologie and Cloud-Native Engineering."
+			"body": "Softwarearchitekt u Mitgründer. Spezialisert uf Microservices, Container-Technologie and Cloud-Native Engineering."
 		},
 		"lena": {
 			"catchphrase": "Head in the Clouds",
-			"body": "Software Engineer und Mitgründerin. Spezialisiert uf Cloud-Dienste, verteilti System und Web-Entwicklig mit moderne Technologie."
+			"body": "Software Engineer u Mitgründerin. Spezialisiert uf Cloud-Dienste, verteilti System u Web-Entwicklig mit moderne Technologie."
 		},
 		"paulina": {
 			"catchphrase": "Cloud Whisperer",
-			"body": "DevOps und Cloud Solution Architektin. Begeisteret vo Outomatisierig. Spezialisiert uf Systemintegration, CI/CD und Cloud-Native Lösige."
+			"body": "DevOps u Cloud Solution Architektin. Begeisteret vo Outomatisierig. Spezialisiert uf Systemintegration, CI/CD u Cloud-Native Lösige."
 		},
 		"john": {
 			"catchphrase": "Cloud Wrangler",
 			"body": "Wouche Büezer. Vou wändig u wouchegängig. Spezialischt für Sofort Lifer Prozäss, Infrastruktur aus Beschribig u Wouche Bouerei."
 		},
 		"denis": {
 			"catchphrase": "Cloud Composer",
-			"body": "Software Engineer. Spezialisiert uf verteilti System, Microservices, event-driven Architekture und Container Technologie."
+			"body": "Software Engineer. Spezialisiert uf verteilti System, Microservices, event-driven Architekture u Container Technologie."
 		},
 		"qasim": {
 			"catchphrase": "☁boy 🤠",
-			"body": "Software Engineer. Begeisteret vo moderne Technologie und Sprache. Immer uf dr Suechi nach em geeignetste Tool i dr Schublade."
+			"body": "Software Engineer. Begeisteret vo moderne Technologie u Sprache. Immer uf dr Suechi nach em geeignetste Tool i dr Schublade."
+		},
+		"philippe": {
+			"catchphrase": "Software Wizzard",
```

Comment:
`Wizard` is spelled with one `z`. Is the word `Wizzard` intentional?

Reply:
I assume it's intentional, yes :sweat_smile: We should ask Phil tomorrow.